api/google_api/server.js

```diff
@@ -3,7 +3,8 @@ const { SceHttpServer } = require('../util/SceHttpServer');
 function main() {
   const API_ENDPOINTS = [
```

Comment:
btw this should work the same if you do
```suggestion
  const API_ENDPOINTS = __dirname + `/routes`
```

as it automatically grabs all the files in that folder